packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -62,18 +75,39 @@ export const InboundLineEdit: FC<InboundLineEditProps> = ({
         onChangeItem={onChangeItem}
       />
       <Divider margin={5} />
-      {wrappedInbound && (
+      {wrappedInbound ? (
         <TabContext value={currentTab}>
-          <TabList
-            value={currentTab}
-            centered
-            onChange={(_, v) => setCurrentTab(v)}
-          >
-            <Tab value={Tabs.Batch} label={Tabs.Batch} />
-            <Tab value={Tabs.Pricing} label={Tabs.Pricing} />
-          </TabList>
+          <Box flex={1} display="flex" justifyContent="space-between">
+            <Box flex={1} />
+            <Box flex={1}>
+              <TabList
+                value={currentTab}
+                centered
+                onChange={(_, v) => setCurrentTab(v)}
+              >
+                <Tab value={Tabs.Batch} label={Tabs.Batch} />
+                <Tab value={Tabs.Pricing} label={Tabs.Pricing} />
+              </TabList>
+            </Box>
+            <Box flex={1} justifyContent="flex-end" display="flex">
+              <BaseButton
+                color="primary"
+                variant="outlined"
+                onClick={onAddBatch}
+              >
+                {t('label.add-batch')}
+              </BaseButton>
+            </Box>
+          </Box>
 
-          <TableContainer sx={{ height: 400 }}>
+          <TableContainer
+            sx={{
+              height: isMediumScreen ? 300 : 400,
+              marginTop: 2,
+              backgroundColor: theme => alpha(theme.palette.gray.pale, 0.5),
```

Comment:
Nice! I really like what you've done with all the styles - a lot! Thanks heaps

I think it works really well when the batch column width is the same between the tabs - though I think it'll be hard to keep that when the tabs have different numbers of columns. I think a `maxWidth` prop on the column might work

I had also been thinking about two other features for columns, since we're here:  the 'fixed first coumn' style, with the item names which would look cool on the detail view tables. Also was thinking about how to add some sort of 'icon'/'component' in between columns - some uses cases would be like adding a `x` and `=` between "# packs", "pack size" and "unit quantity" - and a 'divider' line between sets of columns